docker/worker.dockerfile

```diff
@@ -1,5 +1,6 @@
 ARG FAASM_VERSION
-FROM faasm/base:${FAASM_VERSION}
+ARG FAASM_SGX_MODE
+FROM faasm/base${FAASM_SGX_MODE:+-sgx}:${FAASM_VERSION}
```

Comment:
If `FAASM_SGX_MODE` is not set, leave empty, if it is set, use `-sgx`. [Link to bash wiki.](https://wiki.bash-hackers.org/syntax/pe#use_an_alternate_value)